src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,141 @@
+package duke;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * This is a class that is responsible for storing the tasks into Duke.
+ */
+public class Storage {
+    private final String homeDir;
+    public static ArrayList<Task> taskList;
+
+    Storage() {
+        this.homeDir = System.getProperty("user.dir");
+        if (taskList == null) {
+            taskList = new ArrayList<>();
+        }
+    }
+
+    void load() {
```

Comment:
this method seems to be quite nested, maybe you can consider taking the `switch `statement out into a sub-method